components/resource_control/src/resource_group.rs

```diff
@@ -51,14 +55,35 @@ pub enum ResourceConsumeType {
 }
 
 /// ResourceGroupManager manages the metadata of each resource group.
-#[derive(Default)]
 pub struct ResourceGroupManager {
     resource_groups: DashMap<String, ResourceGroup>,
     registry: RwLock<Vec<Arc<ResourceController>>>,
 }
 
 impl ResourceGroupManager {
-    fn get_ru_setting(rg: &ResourceGroup, is_read: bool) -> u64 {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Self {
+        let manager = Self {
+            resource_groups: Default::default(),
+            registry: Default::default(),
+        };
+
+        // init the default resource group by default.
+        let mut default_group = PbResourceGroup::new();
+        default_group.name = DEFAULT_RESOURCE_GROUP_NAME.into();
+        default_group.priority = MEDIUM_PRIORITY;
+        default_group.mode = GroupMode::RuMode;
+        default_group
```

Comment:
the `default_group` will create in pd side, so tikv will automatically add the group. why do we need add this?

Reply:
The ResourceManage depend on the default group's exists to handle fallback logic. Since the watch pd logic is async, it's possible that the initialization don't finish before some task has come.